.gitignore

```diff
@@ -0,0 +1,25 @@
+node_modules
```

Comment:
Is there a specific reason to put all code changes into a single commit?

Reply:
No. Sorry.
I had made a big to create a local project instead of forking origin repository. 
Finally I moved source code into forked project and commit all of them

Reply:
Got it